modules/arm_plugin/src/transformations/arm_optimizations.cpp

```diff
@@ -277,9 +277,9 @@ bool ArmPlugin::pass::ArmOptimizations::run_on_function(std::shared_ptr<ov::Mode
         manager.register_pass<ov::pass::GraphRewrite>()->add_matcher<pass::ConvertInterpolate>();
         manager.register_pass<ov::pass::GraphRewrite>()->add_matcher<pass::ConvertMVN>();
         manager.register_pass<ov::pass::GraphRewrite>()->add_matcher<pass::ConvertReorgYolo>();
+        manager.register_pass<ov::pass::GraphRewrite>()->add_matcher<pass::ConvertMaxPoolV8>();
```

Comment:
This changes looks similar to this [transformation](https://github.com/openvinotoolkit/openvino/blob/master/src/common/transformations/src/transformations/op_conversions/convert_maxpool_downgrade.cpp). Maybe reuse it?
```cpp
manager.register_pass<ov::pass::GraphRewrite>()->add_matcher<ngraph::pass::ConvertMaxPool8ToMaxPool1>();
```

Reply:
Thanks. I missed the standard one. That's exactly what is required